services/housetables/src/main/java/com/linkedin/openhouse/housetables/dto/mapper/UserTableVersionMapper.java

```diff
@@ -15,11 +15,21 @@
  */
 @Mapper(componentModel = "spring")
 public class UserTableVersionMapper {
```

Comment:
Can you add the documentation to UserTable APISpec?

```
@Schema(description = "Current Version of the user table.", example = "")
  @JsonProperty(value = "tableVersion")
  private String tableVersion;

```

^ this should include info on `new record should have INTITAL_VERISON`

Reply:
ok.